api/v1/paths/onbase/{osuId}.js

```diff
@@ -0,0 +1,55 @@
+const appRoot = require('app-root-path');
+const _ = require('lodash');
+
+const onBaseDao = require('../../db/oracledb/onbase-dao');
+
+const { errorBuilder, errorHandler } = appRoot.require('errors/errors');
+const { openapi: { paths } } = appRoot.require('utils/load-openapi');
+
+/**
+ * @summary Get an OnBase records for a person
+ */
+const get = async (req, res) => {
+  try {
+    const { osuId } = req.params;
+    const result = await onBaseDao.getOnBase(osuId);
+    return res.send(result);
+  } catch (err) {
+    if (err.statusCode === 404) {
+      const errors = _.split(err.message, '|');
+      return errorBuilder(res, 404, errors);
```

Comment:
When handling a 404, won't this function expect a single `detail` instead of an array?